skyportal/schema.py

```diff
@@ -76,21 +76,43 @@ def setup_schema():
                     "Model class ends with 'Schema'"
                 )
 
-            class Meta(object):
+            class MetaExcludeID(object):
```

Comment:
How about using a function to produce these?  Something like:

```
def schema_class_factory(excludes=None):
    ...
    return type(...)
```

Reply:
Done! Check it out...